recipe/meta.yaml

```
@@ -1,49 +1,56 @@
 {% set name = "lancedb" %}
-{% set version = "0.5.7" %}
+{% set version = "0.9.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lancedb-{{ version }}.tar.gz
-  sha256: 878914b493f91d09a77b14f1528104741f273234cbdd6671be705f447701fd51
+  url: https://github.com/lancedb/lancedb/archive/python-v{{ version }}.tar.gz
+  sha256: 52646a68839974d5d5bc3d5564d576c0b089b1d0691b3e6f6ab9e7d7b74ad787
+  patches:
+    - ratelimiter.patch
 
 build:
-  entry_points:
-    - lancedb = lancedb.cli.cli:cli
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
+  skip: true  # [py<=38]
 
 requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ stdlib("c") }}
+    - {{ compiler("rust") }}
+    - cargo-bundle-licenses
+    - pkg-config
+    - protobuf
```

You probably only want `protoc` here.
```suggestion
    - libprotobuf
```